synapse/push/httppusher.py

```diff
@@ -75,6 +75,9 @@ def __init__(self, hs, pusherdict):
         self.failing_since = pusherdict["failing_since"]
         self.timed_call = None
         self._is_processing = False
+        self._push_group_unread_count_by_room = (
```

Comment:
This can just be called `_group_unread_count_by_room` I think since this is *in* the pusher I think this is fairly clear?

Reply:
True! I just default to `self._option_name` :P